decentraland/unity-renderer
# fix: Camera flash after accepting terms

## What does this PR change?

Fixes the first case of #1615.  All the other cases had already been solved in #2972 

## How to test the changes?

1. Start with a fresh copy of DCL
2. Accept terms and conditions
3. Check that the terms and condition canvas stays on until the loading screen fully fades in
4. Start the tutorial normally

Its also a good opportunity to confirm that the loading screen teleports work correctly for all the cases of #1615. For all the following cases, there should be no flash after teleporting.

1. Teleport from chat
2. Teleport from the Explore panel
3. Teleport from map
4. Teleport to any event from the Jump In button present on Genesis Plaza

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
